scripts/jenkins/gcc.groovy

```diff
@@ -76,11 +76,30 @@ image.inside(defaultDockerArgs) {
             keepDir: true,
             script: this
         )
-        build.linux(script: this)
+        build.linux(script: this, target: 'package doc')
     }
 }
 
 stage('Post (Linux-Docker)') {
     post.publishTestReports 'build/Testing/**/*.xml', 'build/Tests/testrunner.xml'
+
+    publishHTML(target: [allowMissing: false, alwaysLinkToLastBuild: true,
+        keepAll: true, reportDir: 'build/docs', reportFiles: 'index.html',
+        reportName: 'Doxygen'])
+    step([$class: 'WarningsPublisher', canResolveRelativePaths: false,
+        messagesPattern: """
+            .*DOT_GRAPH_MAX_NODES.
+            .*potential recursive class relation.*""",
+        parserConfigurations: [[parserName: 'Doxygen', pattern:
+        'build/DoxygenWarnings.log']], unstableTotalAll: '0'])
+
+    if (helper.isOriginMaster(this)) {
+        sshagent(credentials: ['www-data_jenkins']) {
+            sh 'rsync -a --delete --stats -e "ssh -o StrictHostKeyChecking=no"' +
```

Comment:
Is it wise to put `StrictHostKeyChecking=no` here? In this case it might not be a problem, because we just copy some public data there, probably using public key authentication. But in general host keys should be checked. If editing the usual known_hosts file is not an option, there also the possibility to specify a `-oUserKnownHostsFile` on the command line.